docs/content/python/reference/frameworks/flask.md

```diff
@@ -113,6 +113,31 @@ a failed authorization will return a \`\`403 Forbidden\`\` response for the curr
 request.** This can be controlled with
 `set_unauthorized_action()`.
 
+If using specializers in the Polar policy, such as with a `User` class:
+
+```polar
+allow(_actor: User, action: "GET", resource: Request{path: "/"})
+```
+
+... an extra step may need to be taken when using a library or authentication framework (like Flask Login)
+that exposes the current user through `LocalProxy` objects (such as `current_user`).
+
+```python
+from flask_login import current_user
+
+def create_app():
+    app = Flask("app")
+
+    flask_oso.init_app(app)
+    flask_oso.require_authorization(app)
+    # Dereference the current_user LocalProxy
+    flask_oso.set_get_actor(lambda: current_user._get_current_object())
+
+    oso.load_file("authorization.polar")
+    oso.register_class(User)
+    return app
+```
+
```

Comment:
```suggestion
#### Working with `LocalProxy` objects

When using a library that exposes the current user (or other
authorization data) through `LocalProxy` objects, such as [Flask-Login][]'s
`current_user`, you might need to explicitly dereference the proxy
to pass the underlying object to Oso:

[Flask-Login]: https://flask-login.readthedocs.io/en/0.4.1/#flask_login.current_user

```python
from flask_login import current_user

def create_app():
    app = Flask("app")

    flask_oso.init_app(app)
    flask_oso.require_authorization(app)
    # Dereference the current_user LocalProxy
    flask_oso.set_get_actor(lambda: current_user._get_current_object())

    oso.load_file("authorization.polar")
    oso.register_class(User)
    return app
```

By dereferencing the proxy, Oso will use the underlying object when determining
authorization instead of the proxy object.

```